src/leaflike/bookmarks/db.clj

```diff
@@ -0,0 +1,37 @@
+(ns leaflike.bookmarks.db
+  (:require [clojure.java.jdbc :as jdbc]
+            [leaflike.config :refer [db-spec]]
+            [honeysql.core :as sql]
+            [honeysql.helpers :as helpers]))
+
+(defn create
+  [body]
+  (jdbc/execute! (db-spec) (-> (helpers/insert-into :bookmarks)
+                               (helpers/values [body])
+                               sql/format)))
+
+(defn list-all
+  [member_id]
```

Comment:
Not strictly needed, but you could take in params here and destructure it, to make it look regular with the rest